src/core.js

```diff
@@ -2111,6 +2116,9 @@ function _buildSquel(flavour = null) {
     insert: function(options, blocks) {
       return new cls.Insert(options, blocks);
     },
+    replace: function(options, blocks) {
```

Comment:
Surely we can place this inside the `mysql` flavour and thereby get rid of the dummy class in `core`?